### PR TITLE
Feat/upgrade-contracts

### PR DESCRIPTION
**Pull request by Owliie**

### CODE REVIEW

**Comment by Dimo99** [comment]
Write deployment task for `ValidatorAccumulator` contract. And add more test cases to cover all paths

**Review by Dimo99 — APPROVED** [review]

File: beacon-light-client/solidity/contracts/balance_verifier/BalanceVerifierDiva.sol [hunk 1]
```diff
@@ -69,6 +88,10 @@ contract BalanceVerifierDiva is BalanceVerifier, IBalanceVerifierDiva {
     );
   }
 
+  function setAccumulator(address _accumulator) external override onlyOwner {
+    ACCUMULATOR = _accumulator;
+  }
+
```
Comment (Xearty) [inline beacon-light-client/solidity/contracts/balance_verifier/BalanceVerifierDiva.sol:94]: setAccumulator must also set the branch

Comment (Xearty) [reply beacon-light-client/solidity/contracts/balance_verifier/BalanceVerifierDiva.sol:94]: Also, in what cases would you want the ability to set the accumulator. I think this code should execute in the constructor

**Review by Xearty — APPROVED** [review]

**Comment by Xearty** [comment]
I am blind and it lgtm

**Pull request merged**